### DISCUSSION TITLE
不太会用, 尝试sni, 提示需要appid?

### DISCUSSION THREAD

**Discussion by joshatt** [SeaHOH/GotoX#150] [问答/咨询] [answered]
我只需要sni功能, 不需要其他. 
步骤如下:
严格按照教程, 部署了cfw并修改config.ini, 其中在actionfilter.ini------[0-fakecert]中我加了一行:
.abc.de.com = @github.io
不知道这么写对不对
然后运行gotox.vbs  (我这个win7x64电脑上原来安装有python3.7)
按照提示导入证书, 重启chrome
switchyomega设置成8087代理,
访问 mn.abd.de.com  提示需要填写appid. 什么appid?

太菜了, 大家指点一下

**Comment by shpherd** [comment]
XX-NET是使用ID的你到那边看下

**Comment by joshatt** [comment]
> XX-NET是使用ID的你到那边看下

谢谢解答. 看了下,需要国外信用卡绑定. 暂时实现不了

**Comment by SeaHOH** [comment, marked as answer]
> .abc.de.com = @github.io

无需 `@` 符号

> 访问 mn.abd.de.com 提示需要填写appid. 什么appid?

估计是触发临时规则了，当前版本临时规则仍然是 GAE，周末会发正式版就没这个问题。
